internal/hcl/evaluator.go

```diff
@@ -39,7 +39,7 @@ var (
 		"data":     {},
 	}
 
-	sensitiveRegxp = regexp.MustCompile(`_token$|^token_|_secret$|^secret_|_password$|^password_|_username$|^username_|api_key|expiration_date`)
+	sensitiveRegxp = regexp.MustCompile(`^image[_|-]|[_|-]image$|saml[_|-]role$|secrets$|aws[_|-]profile$|key[_|-]id$|secret[_|-]key$|access[_|-]key$|[_|-]token$|^token[_|-]|[_|-]secret$|^secret[_|-]|[_|-]password$|^password[_|-]|[_|-]username$|^username[_|-]|api[_|-]key|expiration[_|-]date`)
```

Comment:
Can we do something like this for better readability?
```suggestion
	sensitiveRegxp = regexp.MustCompile(strings.Join([]string{
		"^image[_|-]",
		"[_|-]image$",
		"saml[_|-]role$",
		"secrets$",
		"aws[_|-]profile$",
		"key[_|-]id$",
		"secret[_|-]key$",
		"access[_|-]key$",
		"[_|-]token$",
		"^token[_|-]",
		"[_|-]secret$",
		"^secret[_|-]",
		"[_|-]password$",
		"^password[_|-]",
		"[_|-]username$",
		"^username[_|-]",
		"api[_|-]key",
		"expiration[_|-]date",
	}, "|"))
```

Reply:
👍